docs/reference/content/driver-scala/tutorials/client-side-encryption.md

```diff
@@ -17,9 +17,6 @@ With field level encryption, developers can encrypt fields client side without a
 configuration or directives. Client-side field level encryption supports workloads where applications must guarantee that 
 unauthorized parties, including server administrators, cannot read the encrypted data.
 
-{{% note class="important" %}}
```

Comment:
The note tag is still needed.  (But no longer class="important").

Added in a extra commit.